manifest.yml

```diff
@@ -13,7 +13,6 @@ services:
 - federalist-production-env
 - federalist-site-wide-error
 env:
-  NEW_RELIC_APP_NAME: federalist-prod
```

Comment:
This isn't used; rather, the value comes from the `federalist-{space}-env` user-provided service.